### ISSUE TITLE
WCAG 3.1.1: <html> element must have a lang attribute (html)

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#788]
## Issue
Multiple pages are missing the `lang` attribute on the `<html>` element. The `lang` attribute specifies the natural language of the page's content and is a WCAG accessibility criterion (see [Language Of Page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html))

To fix the issue, simply change the `<html>` element to `<html lang="en">` (assuming the page is in English).
